sealer/cmd/run.go

```diff
@@ -68,6 +68,7 @@ func init() {
 	runCmd.Flags().StringVarP(&runArgs.PkPassword, "pk-passwd", "", "", "set baremetal server  private key password")
 	runCmd.Flags().StringVarP(&runArgs.PodCidr, "podcidr", "", "", "set default pod CIDR network. example '10.233.0.0/18'")
 	runCmd.Flags().StringVarP(&runArgs.SvcCidr, "svccidr", "", "", "set default service CIDR network. example '10.233.64.0/18'")
+	runCmd.Flags().StringVarP(&runArgs.CustomEnv, "customEnv", "e", "", "set custom environment variables. example 'sealer run -e DashBoardPort=8443 mydashboard:latest'")
```

Comment:
string type is replaced with the slice type（runCmd.Flags().StringSliceVarP）

runArgs.CustomEnv needs to be written to the cluster in `NewApplierFromArgs(args[0], runArgs)` function